my-webxr-app/src/components/DataPointMenu.tsx

```diff
@@ -32,8 +33,11 @@ export default function DataPointMenu(
           {`${selectedDataPoint?.color ?? '-'}\n`}
           x, y, z:
           {' '}
-          {`${selectedDataPoint?.meshProps?.position ?? '-'}\n`}
-          Column X:
+          {`${selectedDataPoint?.meshProps?.position
```

Comment:
'Column X' label no longer shows up so need to add this line back

Reply:
oh whoops, you got it, sorry